reference/import_export/connectors.rst

```diff
@@ -0,0 +1,21 @@
+Akeneo Native Connector
+=================
+
+CSV Format
+-------------
+
+Akeneo Native Connector supports reading and writing CSV files. In order to understand how to customize the connector, please refer to chapters about product import (cf :doc:`/reference/import_export/product-export`) and product export (cf :doc:`/reference/import_export/product-export`).
```

Comment:
imho,
"how to customize" is more related to cookbooks
"understand how it works" is related to reference doc

this connector allow to import/export numerous PIM objects not only products